### PR TITLE
Update bitlocker-overview.md

### PR DESCRIPTION
**Pull request by rafals2**
added more clarity for active partitions

### CODE REVIEW

**Review by illfated — APPROVED** [review]
Looks good and very useful to me. Thank you.

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/information-protection/bitlocker/bitlocker-overview.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The hard disk must be partitioned with at least two drives:
 -   The operating system drive (or boot drive) contains the operating system and its support files. It must be formatted with the NTFS file system.
 -   The system drive contains the files that are needed to load Windows after the firmware has prepared the system hardware. BitLocker is not enabled on this drive. For BitLocker to work, the system drive must not be encrypted, must differ from the operating system drive, and must be formatted with the FAT32 file system on computers that use UEFI-based firmware or with the NTFS file system on computers that use BIOS firmware. We recommend that system drive be approximately 350 MB in size. After BitLocker is turned on it should have approximately 250 MB of free space.
 
-A fixed data volume or removable data volume cannot be marked as an active partition.
+Partition subject to encryption cannot be marked as an active partition (this applies to OS, fixed data and removable data drives).
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-overview.md:77]:
```suggestion
A partition subject to encryption cannot be marked as an active partition (this applies to the operating system, fixed data, and removable data drives).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestions.

**Pull request merged**